openmp/libomptarget/test/offloading/fortran/target_map_common_block.f90

```diff
@@ -80,3 +87,61 @@ subroutine commonblock_simple_with_real
 
 ! CHECK: var3 before target = 12.5
 ! CHECK: var3 after target = 14.5
+
+! -----
+
+subroutine commonblock_simple_to
+  use omp_lib
+  integer :: var4, tmp
+  common var4
+  var4 = 10
+  tmp = 20
+  !$omp target map(to:var4) map(from:tmp)
+    tmp = var4
+    var4 = 20
+  !$omp end target
+  print *, "var4 after target = ", var4
+  print *, "tmp after target = ", tmp
+end subroutine
+
+! CHECK: var4 after target = 10
+! CHECK: tmp after target = 10
+
+! -----
+
+subroutine commonblock_simple_from
```

Comment:
Test `commonblock_simple_to` is already testing the `for` map-type isn't it?

Reply:
Yes, that is correct. Removed this redundant test. Thanks!